python/tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2021 Rikai Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+
+import pytest
+import rikai
+from pyspark.sql import SparkSession
+
+
+@pytest.fixture(scope="session")
+def spark() -> SparkSession:
+    # TODO: load rikai jar from "sbt publishLocal" installed version?
+    return (
+        SparkSession.builder.appName("spark-test")
+        .config(
+            "spark.jars",
+            os.path.join(
+                os.path.dirname(rikai.__file__), "jars", "rikai_2.12-0.0.1.jar"
```

Comment:
is there an easy way to pull the current version number? If it's a hassle, just create a github issue and we can wait til after 0.0.1

Reply:
we can use a `glob` actually.  Do you think we should wait for `sbt publishLocal` to install a `-SNAPSHOT` version?

Reply:
I kind lean to not have this `jars` directory in the future.  Even on a Databricks cluster, the jars will be deployed separately from this `pip install rikai`.